README.md

```diff
@@ -1,5 +1,106 @@
-# Astro Databricks
+<h1 align="center">
+  Astro Databricks
+</h1>
+  <h3 align="center">
+  Affordable Databricks Workflows in Apache Airflow<br><br>
+</h3>
 
-Astro Databricks is an airflow provider created by Astronomer for an optimal databricks experience.
+[![Python versions](https://img.shields.io/pypi/pyversions/astro-providers-databricks.svg)](https://pypi.org/pypi/astro-providers-databricks)
+[![License](https://img.shields.io/pypi/l/astro-providers-databricks.svg)](https://pypi.org/pypi/astro-providers-databricks)
+[![Development Status](https://img.shields.io/pypi/status/astro-providers-databricks.svg)](https://pypi.org/pypi/astro-providers-databricks)
+[![PyPI downloads](https://img.shields.io/pypi/dm/astro-providers-databricks.svg)](https://pypistats.org/packages/astro-providers-databricks)
+[![Contributors](https://img.shields.io/github/contributors/astronomer/astro-providers-databricks)](https://github.com/astronomer/astro-providers-databricks)
+[![Commit activity](https://img.shields.io/github/commit-activity/m/astronomer/astro-providers-databricks)](https://github.com/astronomer/astro-providers-databricks)
+[![CI](https://github.com/astronomer/astro-providers-databricks/actions/workflows/ci.yml/badge.svg)](https://github.com/astronomer/astro-providers-databricks)
+[![codecov](https://codecov.io/gh/astronomer/astro-providers-databricks/branch/main/graph/badge.svg?token=MI4SSE50Q6)](https://codecov.io/gh/astronomer/astro-providers-databricks)
 
-For up-to-date documentation, visit https://astronomer.github.io/astro-providers-databricks/
\ No newline at end of file
+
+**Astro Databricks** is an [Apache Airflow](https://github.com/apache/airflow) provider created by [Astronomer](https://www.astronomer.io/) for an **optimal Databricks experience**.  With the `DatabricksTaskGroup`, Astro Datatricks allows you to run from Databricks workflows without
+the need of running Jobs individually, which can result in **75% cost reduction**.
+
+## Prerequisites
+
+* Apache Airflow >= 2.2
+* Python >= 3.7
+* Databricks account
+* Previously created Databricks Notebooks
+
+## Install
+
+```shell
+pip install astro-providers-databricks
+```
+
+## Quickstart
+
+1. Use pre-existing or create two simple [Databricks Notebooks](https://docs.databricks.com/notebooks/). Their identifiers will be used in step (5). The original example DAG uses: 
+   * `Shared/Notebook_1`
+   * `Shared/Notebook_2`
+
+2. Generate a [Databricks Personal Token](https://docs.databricks.com/dev-tools/auth.html#databricks-personal-access-tokens). This will be used in step (6). 
+
+3. Ensure that your Airflow environment is set up correctly by running the following commands:
+
+    ```shell
+    export AIRFLOW_HOME=`pwd`
+    export AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES": "airflow\\.* astro_databricks\\.*"
+   
```

Comment:
Add note about pickling and Airflow versions under 2.5

Reply:
No longer necessary 🎉 
https://github.com/astronomer/astro-providers-databricks/pull/15